src/Console/Command/ComposerCommand.php

```diff
@@ -38,15 +40,34 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $extra = array_filter($this->getRawTokens($input), fn ($item) => 'composer' !== $item);
+        $extra = explode(' ', implode(' ', $extra));
```

Comment:
IDK why for now, but i needed to do that for running command like `castor composer "init --help"`
But without this line composer output this :
```
Command "init --help" is not defined.
```

Reply:
Which is normal, since encapsulating with '"' will make it an arg, imo you should drop this and for the moment we could accept that it's not possible to use same options in castor / sub commands (it's a global problem).

A fix would be to implement this use case directly into symfony not a find of providing such hacks here as it would break others behavior (like i want a space in a arg which can be common)